modules/data-entry/src/main/frontend/src/dataHomepage/NewFormDialog.jsx

```diff
@@ -232,27 +221,48 @@ function NewFormDialog(props) {
           {isFetching && <div className={classes.newFormTypePlaceholder}><CircularProgress size={24} className={classes.newFormTypeLoadingIndicator} /></div>}
           {progress === PROGRESS_SELECT_QUESTIONNAIRE ?
           <React.Fragment>
-            <Typography variant="h4">Questionnaire</Typography>
-            {questionnaires && relatedForms &&
-              <List>
-                {questionnaires.map((questionnaire) => {
-                  // if selectedSubject has already been set, check if maxPerSubject has been reached for each questionnaire
-                  // atMax returns true if the number of `relatedForms` is greater than the maxPerSubject
-                  let atMax = (relatedForms.length && (relatedForms.filter((i) => (i["@name"] == questionnaire["@name"])).length >= questionnaire?.["maxPerSubject"]));
-                  return (
-                  <SubjectListItem
-                    key={questionnaire["jcr:uuid"]}
-                    onClick={() => {setSelectedQuestionnaire(questionnaire)}}
-                    disabled={isFetching}
-                    selected={questionnaire["jcr:uuid"] === selectedQuestionnaire?.["jcr:uuid"]}
-                    avatarIcon={AssignmentIcon}
-                    >
-                    <div className={`${atMax ? classes.questionnaireDisabledListItem : classes.questionnaireListItem}`}>
-                      <ListItemText primary={questionnaire["title"]} secondary={questionnaire["description"]}/>
-                    </div>
-                  </SubjectListItem>);
-                })}
-              </List>
+            {relatedForms &&
+              <MaterialTable
+                title=""
+                columns={[
+                  { title: 'Questionnaire', field: 'title' },
+                  { title: 'Description', field: 'description' },
+                ]}
+                data={query => {
+                  let url = new URL("/query", window.location.origin);
+                  url.searchParams.set("query", `select * from [lfs:Questionnaire] as n${query.search ? ` WHERE CONTAINS(n, '*${query.search}*')` : ""}`);
+                  url.searchParams.set("limit", query.pageSize);
+                  url.searchParams.set("offset", query.page*query.pageSize);
+                  return fetch(url)
+                    .then(response => response.json())
+                    .then(result => {
+                      return {
+                        data: result["rows"],
+                        page: Math.trunc(result["offset"]/result["limit"]),
+                        totalCount: result["totalrows"],
+                      }}
+                    )
+                }}
+                options={{
+                  search: true,
+                  actionsColumnIndex: -1,
+                  addRowPosition: 'first',
+                  pageSize: 10,
```

Comment:
For consistency with the subjects table, I suggest to use a page size of 5.
```suggestion
                  pageSize: 5,
```

Reply:
MaterialTable defaults to 5 results: removed pageSize option to fallback to that default as that is what the subject table is doing.